### PR TITLE
Workflow for GRASS GIS manuals

### PR DESCRIPTION
**Pull request by anikaweinmann**
Add workflow for GRASS GIS manuals. The workflow uses GRASS 8 by own Dockerfile the workflow is moved to [reusable workflow](https://github.com/mundialis/github-workflows/pull/18).

### CODE REVIEW

File: .github/workflows/grass-manual.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        with:
+          packages: grass grass-dev grass-doc wget make
+          # version: 1.0
+      - name: Install dependencies
+        run: |
+          # apt-get install -y -qq grass grass-dev grass-doc wget make
+          mkdir public
+          # some files are not installed properly, get them online
+          (cd public && wget https://grass.osgeo.org/grass74/manuals/grass_logo.png)
+          (cd public && wget https://grass.osgeo.org/grass74/manuals/grassdocs.css)
```
Comment (neteler) [inline .github/workflows/grass-manual.yml]:
```suggestion
          (cd public && wget https://grass.osgeo.org/grass-stable/manuals/grass_logo.png)
          (cd public && wget https://grass.osgeo.org/grass-stable/manuals/grassdocs.css)
```

File: .github/workflows/grass-manual.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      - name: Create a temporary location (needed for v<7.6)
```
Comment (neteler) [inline .github/workflows/grass-manual.yml]: `<v7.6` is a while ago :) Still needed?

File: .github/workflows/grass-manual.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        run: |
+          grass -c EPSG:4326 grasstmploc -e
+      - name: Complile GRASS GIS addon
```
Comment (neteler) [inline .github/workflows/grass-manual.yml]:
```suggestion
      - name: Compile GRASS GIS addon
```

**Comment by neteler** [comment]
Apparently still GRASS GIS 7 is used here, as the CI log shows:

`/usr/lib/grass78/scripts/g.extension ...`

Maybe a wrong docker image?

File: .github/workflows/grass-manual.yml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      - name: Move manual to public folder
+        run: |
+          ADDON_NAME=$(echo ${GITHUB_REPOSITORY} | cut -d "/" -f 2)
+          # put the compile documentation to the public directory
+          # public name is convention for published web pages
+          cp -r build/docs/html/* public/
+          cp public/${ADDON_NAME}.html public/index.html
+      - name: Reset URL
+        run: |
+          sed -i "s+file:///usr/lib/grass78/docs/html/+https://grass.osgeo.org/grass83/manuals/+g" public/index.html
```
Comment (neteler) [inline .github/workflows/grass-manual.yml]:
```suggestion
          sed -i "s+file:///usr/lib/grass78/docs/html/+https://grass.osgeo.org/grass-stable/manuals/+g" public/index.html
```

Please consider to use `-stable` to point to G8, this always works, also once G8.4 is (soonish) out.

**Review by mmacata — APPROVED** [review]

File: .github/workflows/grass-manual.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+name: GRASS GIS addon manual
+
+on: [push]
```
Comment (mmacata) [inline .github/workflows/grass-manual.yml]: I mentioned this in the github-workflow PR as well: Would it make sense to only run for `main` branch?

**Pull request merged**